evm_loader/test_transaction.py

```diff
@@ -215,8 +215,8 @@ def check_transfers_between_operator_deposit_and_collateral_pool(
         collateral_pool_balance_change = int(collateral_pool_post_balance) - int(collateral_pool_pre_balance)
         print('     collateral_pool_balance_change:', collateral_pool_balance_change)
         if step is Step.Begin:
-            self.assertEqual(operator_balance_change, 0 - fee - NEON_PAYMENT_TO_DEPOSIT - NEON_PAYMENT_TO_TREASURE)
-            self.assertEqual(deposit_balance_change, NEON_PAYMENT_TO_DEPOSIT)
+            self.assertEqual(operator_balance_change, 0 - fee - NEON_PAYMENT_TO_TREASURE)
```

Comment:
Why we don't pay to treasure for the beginning iteration?

Reply:
It is the second transaction. The first one doesn't execute steps.

Reply:
fixed